docs/store.js

```diff
@@ -407,13 +407,13 @@ var LocalCalendarStorage = function(params) {
       
     };
     
-    var jitter = function(func, arg) {
+    var jitter = function(params) {
```

Comment:
So I guess this was my fault for not explaining what I meant further. I meant that the jitter size could be set by a value you pass to the storage at construction. If you needed to pass different values in different settings, then what you've done would make sense, but we don't really need that since all this is doing is simulating random delays.